shell/platform/linux/BUILD.gn

```diff
@@ -61,7 +61,7 @@ if (build_linux_shell) {
     sources = [
       "fl_dart_project.cc",
       "fl_view.cc",
-    ]
+    ] + _public_headers
```

Comment:
This should be:

```
public = _public_headers

sources = [
  "fl_dart_project.cc",
  "fl_view.cc",
]
```
`public` is like `sources`, but means that other GN targets depending on this target are allowed to include those headers. (GN allows enforcement that header dependencies are reflected in target dependencies.)